.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "3.6.4"
+
+install:
+  - pip install -r requirements.txt
+
+script:
+  - python3 manage.py test
+
+after_success:
+  - coveralls
```

Comment:
same here

Reply:
No, it was not, I rebased with develop when I got merge conflicts after you had merged the two pending PR's to develop and so I got the .coveralls.yml file.